ckanext/data_qld/google_analytics/plugin.py

```diff
@@ -21,24 +20,24 @@ class AnalyticsPostThread(threading.Thread):
     def __init__(self, queue):
         threading.Thread.__init__(self)
         self.queue = queue
-        self.ga_collection_url = config.get('ckan.data_qld_googleanalytics.collection_url',
-                                            'https://www.google-analytics.com/collect')
+        self.ga_collection_url = config.get('ckanext.data_qld_googleanalytics.ga4_collection_url',
```

Comment:
where is the ga4 and the api token being added to the url get params?

Reply:
That is in the cookbook.